arezoo/script.js

```diff
@@ -0,0 +1,13 @@
+let strOne = "HelloLadies";
+console.log(strOne[2], strOne[6]);
```

Comment:
2. Access the 2nd and 6th elements of the string. Print the elements to the console. **Make sure you have selected the right index numbers** 

strOne[2] is the 3rd